docs/book/features/orphan_snapshots.md

```diff
@@ -0,0 +1,71 @@
+This API also takes optional parameter snapshotPrefix, which indicates the prefix used in the snapshot description.
+
+Please note if there are large number of orphan snapshots in the system or if there's a slowness in vCenter networking/storage, the orphan snapshot deletion may take longer. That is why, orphan snapshot deletion operation is performed asynchronously.
+It creates a job to delete these orphan snapshots in the background and returns the job Id to user, the status of this job can be retrieved using `getjobstatus` API.
+
+
+Apart from APIs provided for detection/deletion of orphan snapshots, there are separate APIs to list/delete snapshots of a specific volume or to delete an individual snapshot.
+
+3. *GET /volumes/{volumeId}/snapshots*
+This API lists all the snapshots for a specific volume. It takes required parameters datacenter and datastore name to which volume and snapshots belong to.
+This API also takes optional parameter snapshotPrefix which indicates the prefix used in snapshot description. If snapshot prefix is not specified, then it lists all snapshots of a volume. Otherwise it lists only those snapshots of volume whose description’s prefix matches with the specified snapshot prefix.
+
+4. *DELETE /volumes/{volumeId}/snapshots*
+This API deletes all snapshots of a specific volume. Similar to GET API, it takes required parameters datacenter and datastore and optional parameter snapshotPrefix.
+Deletion of all snapshots of a volume is performed asynchronously. It creates a job to delete these snapshots in the background and returns the job Id to user, the status of this job can be retrieved using `getjobstatus` API.
+
+5. *DELETE /volumes/{volumeId}/snapshots/{snapshotId}*
+This API deletes a specific snapshot belonging to a volume. It takes required parameters datacenter and datastore name to which volume and snapshot belongs to.
+Deletion of a snapshot is performed asynchronously. It creates a job to delete this snapshot in the background and returns the job Id to user, the status of this job can be retrieved using `getjobstatus` API.
+
+Two APIs are being offered to check the status of the snapshot deletion job (A job corresponds to a single invocation of the API, that can have multiple tasks corresponding to separate snapshot deletion).
+i. *GET /getjobstatus*
+This API returns the current status of the job. A job can be in one of the following status:
+    * Queued - Job has been created but hasn't started processing.
+    * Running - Job is currently executing.
+    * Success - Job has completed successfully with all tasks succeeding.
+    * Error - Job ran but some or all of its tasks failed.
+ii. *GET /waitforjob*
+This is a blocking API that waits for job to be successful or fail. Unlike getjobstatus API, this will wait for the job to finish before returning the job result response.
+
```

Comment:
I don't see supported vSphere version, K8s version, vSphere CSI driver version, and Velero vSphere Plugin version mentioned in the doc.  We should add that.

Reply:
I have added supported versions section, please check if it looks ok. I have asked Ravi as well to take a look.